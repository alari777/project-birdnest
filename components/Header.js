const Header = ({ text }) => {
    return (
        <header>
            PROJECT BIRDNEST: MONITORING SYSTEM
        </header>
    )
}

export default Header;