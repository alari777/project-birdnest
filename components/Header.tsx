import { FC } from 'react';

const Header: FC = () => {
  return <header>PROJECT BIRDNEST: MONITORING SYSTEM</header>;
};

export default Header;
