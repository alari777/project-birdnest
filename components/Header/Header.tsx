import { FC } from 'react';
import styles from './Header.module.scss';

const Header: FC = () => {
  return (
    <header className={styles.header}>
      PROJECT BIRDNEST: MONITORING SYSTEM
    </header>
  );
};

export default Header;
