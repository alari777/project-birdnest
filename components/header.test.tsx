import { render, screen } from '@testing-library/react';
import Header from './Header';

it('should render header', () => {
  render(<Header />);
  const header = screen.getByText(/PROJECT BIRDNEST: MONITORING SYSTEM/i);
  expect(header).toBeInTheDocument();
  expect(header).toMatchSnapshot();
});
