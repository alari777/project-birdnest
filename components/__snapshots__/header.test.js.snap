// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<header>
  PROJECT BIRDNEST: MONITORING SYSTEM
</header>
`;
